src/main/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/viewmodel/JobView.java

```diff
@@ -28,20 +30,20 @@
         put(ABORTED,   "failing");  // if someone has aborted it then something is clearly not right, right? :)
     }};
 
-    public static JobView of(Job<?, ?> job) {
-        return new JobView(job, new BuildAugmentor(), RelativeLocation.of(job), new Date());
+    public static JobView of(Job<?, ?> job, Config config) {
```

Comment:
I think many of these methods exist _only_ for test constructs so shouldn't live in the prod code at all. I won't move them but will add a method to the test pack like you say.